docs/developer-docs/integrations/sns/managing/making-proposals.md

```diff
@@ -0,0 +1,98 @@
+# SNS proposals
+
+## Overview
+
+To manage an SNS, developers and community need to understand how proposals work, how they can be submitted, voted on, and what effect they have. This article explains the different types of proposals and how to work with them.
```

Comment:
```suggestion
To manage an SNS, the SNS community needs to understand how proposals work, how they can be submitted, voted on, and what effect they have. This article explains the different types of proposals and how to work with them.
```
(at this point the devs are equal to other community members in teh view of the DAO, this is why I would maybe not single them out)